diff --git a/cube2/src/web/Makefile b/cube2/src/web/Makefile
index 6b53252..c7ee8b9 100644
--- a/cube2/src/web/Makefile
+++ b/cube2/src/web/Makefile
@@ -5,7 +5,7 @@ CXX = $(EMSCRIPTEN)/emcc
 EMCONFIGURE = $(EMSCRIPTEN)/emconfigure
 EMMAKE = $(EMSCRIPTEN)/emmake
 
-CXXFLAGS= -O3 -s USE_ZLIB=1
+CXXFLAGS= -O3 -g -s USE_ZLIB=1
 override CXXFLAGS+= -Wall -fsigned-char -fno-exceptions -fno-rtti
 
 PLATFORM= emscripten
